sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/HealthApiClient.cs

```diff
@@ -42,13 +42,13 @@ public HealthApiClient(string endpoint, WebPubSubServiceClientOptions options =
             options ??= new WebPubSubServiceClientOptions();
 
             _clientDiagnostics = new ClientDiagnostics(options);
-            _pipeline = HttpPipelineBuilder.Build(options, new HttpPipelinePolicy[] { new LowLevelCallbackPolicy() }, Array.Empty<HttpPipelinePolicy>(), new ResponseClassifier());
```

Comment:
@JialinXin - could you please verify that the changes in WebPubSub area are expected?

Reply:
This update is to adopt new features in Core.